src/render_systems.rs

```diff
@@ -21,9 +23,16 @@ use bevy::{
     utils::HashMap,
 };
 
-/// Extracted Egui settings.
-#[derive(Resource, Deref, DerefMut, Default)]
-pub struct ExtractedEguiSettings(pub EguiSettings);
+/// The extracted version of [`EguiManagedTextures`].
+#[derive(Debug, Resource)]
+pub struct ExtractedEguiManagedTextures(pub HashMap<(Entity, u64), Handle<Image>>);
```

Comment:
Let's turn this `(Entity, u64)` tuple into a struct, named fields will help to understand what `Entity` and `u64` are related to.